docs/conf.py

```diff
@@ -72,18 +72,21 @@ def check_if_v046():
     'sphinx_multiversion'
 ]
 
+source_suffix = ['.rst', '.md']
```

Comment:
interesting, are we mixing the two formats? :D

Reply:
I couldn't quite remember why I did this a few months ago, so I removed the `.md`  portion and rebuilt. Turns out nothing much happens... so I'll remove it.